tardis/model.py

```diff
@@ -207,7 +207,7 @@ def calculate_j_blues(self, init_detailed_j_blues=False):
             logger.info('Calculating J_blues for radiate_rates_type=detailed')
 
             self.j_blues = pd.DataFrame(
-                self.j_blue_estimators.transpose() *
+                self.j_blue_estimators *
                 self.j_blues_norm_factor.value,
```

Comment:
Guys - let's not get sidetracked here - we want to fix the detailed mode! Let's merge and not get bugged down in style questions (which we also have not yet been formalised!)